ReadCourseData.py

```diff
@@ -0,0 +1,153 @@
+
+def from_raw_to_list(course_raw):
+	course_list, department_list = [], []
+
+	for department in course_raw['2018 Winter De Anza']['CourseData']:
+		temp_dept = Department(department)
+		temp_dept = read_course(course_raw, course_list, department, temp_dept)
+		department_list.append(temp_dept)
+
+
+	return course_list, department_list
+
+def read_course(course_raw, course_list, department, temp_dept):
+	for c in course_raw['2018 Winter De Anza']['CourseData'][department]:
+			temp_course = Course(UID = c['CRN'],crn = c['CRN'], course_num = c['Crse'], 
+							 section_num = c['Sec'], campus = c['Cmp'], num_credit = c['Cred'],
+							 course_title = c['Title'], days = c['Days'], startTime = c['Time'][:8], 
+							 endTime = c['Time'][9:], cap = c['Cap'], 
+							 wl_cap = c['WL Cap'],instructor_name = c['Instructor'],startDate = c['Date'][:5], 
+							 endDate = c['Date'][6:],location = c['Location'],attribute = c['Attribute'])
+			course_list.append(temp_course)
+			course_ineach_dept = '{0} {1} {2}'.format(temp_dept.deptName, temp_course.course_num, temp_course.course_title) 
+			if course_ineach_dept not in temp_dept.courses:
+				temp_dept.courses.append(course_ineach_dept)
+	return temp_dept
+
+
+def course_obj_to_dict(course):
+	temp_course = {
+			'UID' : course.UID,
+			'crn': course.crn,
+			'course_num' : course.course_num,
+			'section_num' : course.section_num,
+			'campus' : course.campus,
+			'num_credit' : course.num_credit,
+			'course_title' : course.course_title,
+			'days' : course.days,
+			'start_time' : course.startTime,
+			'end_time' : course.endTime,
+			'cap' : course.cap,
+			'wl_cap' : course.wl_cap,
+			'instructor_name' : course.instructor_name,
+			'start_date' : course.startDate,
+			'end_date' : course.endDate,
+			'location' : course.location,
+			'attribute' : course.attribute,
+		}
+	return temp_course
+
+def dept_obj_to_dict(dept):
+	temp_dept = {
+			'department_name' : dept.deptName,
+			'course_list' : dept.courses
+		}
+	return temp_dept
+
+def get_db():
+	env_path = Path('.') / '.env'
+	load_dotenv(dotenv_path=env_path)
+	username = os.getenv('Mongo_User')
+	password = os.getenv('Mongo_Password')
+	db_name = 'yifeil_test'      #os.getenv('Mongo_DBName') #just tesing the code
+	client = MongoClient('mongodb+srv://' + username +':' + password + '@fhdatimedb-jjsjm.mongodb.net/test?retryWrites=true&w=majority')
+	return client.get_database(db_name)
+
+def insert_data(course_list, dept_list):
+	db = get_db()
+	tc = db.new_test_courses  #tc for test_course, just testing the code
+	td = db.new_test_depts	#td for test_dept., just testing the code
+
+
+	for course in course_list:
+		temp_course = course_obj_to_dict(course)
+		tc.insert_one(temp_course)
+
+	for dept in dept_list:
+		temp_dept = dept_obj_to_dict(dept)
+		td.insert_one(temp_dept)
+
+
+
+
+def main():
+
+	filename = input('Please enter file path (Example E:\\Personal_Workflow\\FHDA\\DBImport\\2018_Winter_De_Anza_courseData.json)\n')
```

Comment:
nit: it might be better to avoid asking for user's input like this. rather, having a config file might be better. a config file will be very helpful especially when you have lots of configurations to set up later.